website/docs/Installation/windows-install.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 4
+---
+
+# Windows
+## 1. Installing Podman Desktop on Windows
+
+In order to install the latest Podman Desktop application for Windows, visit the [Downloads](/downloads/windows) section of this website to download the .exe file.
+
+Simply download the file from the [Downloads](/downloads/windows) section and open it in your Desktop to install Podman Desktop.
+
+## 2. Installing Podman (if not already present)
+
+If you don't have Podman installed in your Windows computer, Podman Desktop will prompt you to do so as soon as you open the application. With the latest update, Podman Desktop will be able to install and configure Podman once you click on the 'Install' button on the home page.
+
+![img1](img/homescreen.png)
+
+### 3. Initialize & Start the Podman Machine
+
+In order to initialize a new Podman Machine, go to Preferences on the menu present in the left-side of the application. Under Resources, you will find Podman. Clicking on it shall load the configuration settings for the machine. Enter the values that deem fit for your purpose and click on the "Create" button.
+
+![img2](img/create.png)
+
+Once the machine is created, you can click on the Start button in the Machine Settings to start the Machine.
+
+![img3](img/machine.png)
+
+After which, you will be all set to use Podman Desktop on Windows!
```

Comment:
with latest update, you can do that from the welcome page by using toggle.

Maybe you can also say that it's possible to do it with command line for advanced usage like with:

```
$ podman machine init
$ podman machine start
```